src/main/webapp/i18n/de/lectureUnit.json

```
@@ -130,7 +130,8 @@
                 "version": "Version",
                 "fileRequiredValidationError": "Du musst eine Datei zum uploaden auswählen.",
                 "chooseFile": "Datei auswählen",
-                "fileLimitations": "Unterstützte Formate: \".png\", \".jpg\", \".jpeg\", \".svg\", \".pdf\", \".zip\". Max. Dateigröße 10 mb"
+                "fileLimitation": "Max. Dateigröße 10 mb",
```

```suggestion
                "fileLimitation": "Max. Dateigröße 10 MB",
```
Is it 10 megabytes or 10 megabits? I assume the first. In any case the `m` should be uppercase. Same for the English translation string.

---

Thanks, fixed